kontrol-service/engine/flow/render.go

```diff
@@ -232,12 +232,25 @@ func getDestinationRule(serviceID string, services []*resolved.Service, namespac
 	// if we do that then the render work around isn't necessary
 	subsets := lo.UniqBy(
 		lo.Map(services, func(service *resolved.Service, _ int) *v1alpha3.Subset {
-			return &v1alpha3.Subset{
+
+			newSubset := &v1alpha3.Subset{
 				Name: service.Version,
 				Labels: map[string]string{
 					"version": service.Version,
 				},
 			}
+
```

Comment:
Is this disabling TLS for all dev (non-prod) services regardless of whether or not telepresence is used? Wondering if later would be better to narrow it to specific services

Reply:
both ticket and comment created